### DISCUSSION TITLE
Bios fails to download ipxe.efi

### DISCUSSION THREAD

**Discussion by eriklindstein** [ipxe/ipxe#519] [Q&A]
We been running a httpboot solution with ipxe and a fitlet pc from compulab.

Things been working perfectly for a long time, then we install a new server and the bios on the fitlet fails to download ipxe.
I see in my web server log files that bios do a HEAD ipxe.efi but then no GET. Just stops.

I can download the files perfectly with any other web client, it's just the bios that fails. If I place ipxe.efi on my Amazon server and point there it works fine, ipxe can then download the rest of the needed files from my local web server. I also have a local lte router with web server and if I put ipxe.efi there it also works. 

First I tried the setup on Ubuntu 20.04 with the problem, I then downgraded to 18.10 and it worked just fine again. 

Now a couples of weeks later it's the same when we deploy a new server with 18.10 don't fetch the file..

I think it's some incompatible issue with the ip stack in the bios and some update made to the Linux ip stack. But it's very hard to understand where to dig in deeper.

Any one seen similar issues?

**Comment by NiKiZe** [comment]
So this is an issue with your servers or machines efi firmware, since the issue is before iPXE even gets started. Correct?

**Reply by eriklindstein** [reply]
Yes, ipxe works perfectly.
Just hoping someone perhaps seen similar issues.
